AssetHelper/CatalogTools/Catalog.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using UnityEngine.AddressableAssets.ResourceLocators;
+using UnityEngine.ResourceManagement.ResourceLocations;
+using UnityEngine.ResourceManagement.Util;
+
+namespace Silksong.AssetHelper.CatalogTools
```

Comment:
Any chance you can turn this into a file scoped namespace? In my version of visual studio, if you put a semicolon after the namespace name it de-indents the rest of the file automatically (not sure if that's the same in your IDE)